### PR TITLE
Connector-Todo-Notification Csharp Sample Migrated to 6.0

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/TeamsAppPackages/manifest.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
   "manifestVersion": "1.12",
   "version": "1.0.0",
-  "id": "d61ca023-5aa4-423c-8fd6-1a29d332616e",
+  "id": "c5f12998-f395-4eaa-ad7c-5f5bab2d47bb",
```
Comment (Pawan-MSFT) [inline samples/connector-todo-notification/csharp/TeamsToDoAppConnector/TeamsAppPackages/manifest.json:5]: Should be change <MenifestID> ?

File: samples/connector-todo-notification/csharp/README.md [hunk 2]
```diff
@@ -24,8 +24,15 @@ This application simulates a real task management system and allows users to cre
 ## Prerequisites
 The minimum prerequisites to run this sample are:
 * The latest update of Visual Studio. You can download the community version [here](http://www.visualstudio.com) for free.
-* An Office 365 account with access to Microsoft Teams, with [sideloading enabled](https://msdn.microsoft.com/en-us/microsoft-teams/setup).
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/README.md:27]: Why did you remove this line?

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Controllers/ConnectorController.cs [hunk 3]
```diff
@@ -1,8 +1,8 @@
-﻿using System;
-using System.Linq;
-using System.Threading.Tasks;
-using System.Web.Mvc;
+﻿using Bogus;
+using Microsoft.AspNetCore.Mvc;
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Controllers/ConnectorController.cs:2]: We can move the code outside this folder and can remove it:
image

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      "applicationUrl": "http://localhost:3978",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "TeamsToDoAppConnector": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "launchUrl": "Index.html",
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Properties/launchSettings.json:16]: Make sure you have this file exist in your code: Index.html

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Views/Home/Index.cshtml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+﻿@{
+    ViewData["Title"] = "Home Page";
+}
+
+<div class="text-center">
+    <h1 class="display-4">Welcome</h1>
+    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Views/Home/Index.cshtml:7]: Rename it as per your app name

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/bootstrap/LICENSE [hunk 6]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2011-2021 Twitter, Inc.
+Copyright (c) 2011-2021 The Bootstrap Authors
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/bootstrap/LICENSE:10]: I think, this file is not required.

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/jquery-validation-unobtrusive/LICENSE.txt [hunk 7]
```diff
@@ -0,0 +1,12 @@
+Copyright (c) .NET Foundation. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+these files except in compliance with the License. You may obtain a copy of the
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/jquery-validation-unobtrusive/LICENSE.txt:4]: This file is not required?

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/jquery-validation/LICENSE.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+Copyright Jörn Zaefferer
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/jquery-validation/LICENSE.md:12]: This file is not required?

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/jquery/LICENSE.txt [hunk 9]
```diff
@@ -0,0 +1,36 @@
+Copyright JS Foundation and other contributors, https://js.foundation/
+
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/jquery/LICENSE.txt:2]: This file is not required?

File: samples/connector-todo-notification/csharp/assets/sample.json [hunk 10]
```diff
@@ -9,7 +9,7 @@
       "NOTE: This has no relationship with the Microsoft To Do product. A simple in-memory database holds the to do items."
     ],
     "creationDateTime": "2018-01-16",
-    "updateDateTime": "2022-06-21",
+    "updateDateTime": "2022-12-03",
```
Comment (ChetanSharma-msft) [inline samples/connector-todo-notification/csharp/assets/sample.json:12]: Check these warnings:
image

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Properties/launchSettings.json (hunk 4 above)
Comment (Mohammed-MSFT) [reply samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Properties/launchSettings.json:16]: Yes, exists in file.

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/bootstrap/LICENSE (hunk 6 above)
Comment (Mohammed-MSFT) [reply samples/connector-todo-notification/csharp/TeamsToDoAppConnector/wwwroot/lib/bootstrap/LICENSE:10]: ok

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Controllers/ConnectorController.cs (hunk 3 above)
Comment (Mohammed-MSFT) [reply samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Controllers/ConnectorController.cs:2]: ok

File: samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Views/Home/Index.cshtml (hunk 5 above)
Comment (Mohammed-MSFT) [reply samples/connector-todo-notification/csharp/TeamsToDoAppConnector/Views/Home/Index.cshtml:7]: Fixed

**Comment by ChetanSharma-msft** [comment]
**Executed build pipeline:** https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3628883936
**Result:**

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

**Review by Wajeed-msft — APPROVED** [review]
Please add this solution in build pipeline.
